tests/MemcachedTest.php

```diff
@@ -1,32 +1,44 @@
 <?php
+
 namespace Yiisoft\Cache\Tests;
 
 use Yiisoft\Cache\Cache;
 use Yiisoft\Cache\CacheInterface;
 use Yiisoft\Cache\Memcached;
+use Yiisoft\Cache\MemcachedServer;
 
 /**
  * Class for testing memcached cache backend.
+ * If memcached server is not running on localhost at port 11211 set environment variables MEMCACHED_HOST and
+ * MEMCACHED_PORT to use different host and port of the memcached server in the tests
  * @group memcached
  * @group caching
  */
 class MemcachedTest extends CacheTest
 {
+    public static $MEMCACHED_HOST;
+
+    public static $MEMCACHED_PORT;
```

Comment:
Why these properties does not follow naming conventions for properties?

Reply:
Yes, these should be memcachedHost and memcachedPort. Also I don't see why these should be public.